source/components/reveal/HxReveal.less

```diff
@@ -1,8 +1,10 @@
+@import "../HxElement";
+
 :host {
   display: block;
 
-  &([open]) > #content {
-    display: block;
+  &([open]) #content {
```

Comment:
Because we originally had a `<div>` wrapper in the shadow dom of HxReveal, the polyfill would insert that div into the rewritten markup (resulting in multiple `#content.hx-reveal` elements). This meant that we had to increase specificity of the selector to only apply to direct children (using `>` in the css selector).  

Now that we've unwrapped the DIV in the shadow dom, we no longer need the `>` in the css selector, because the polyfill will just replace `<slot>` with the appropriate content. The light dom css in `reveal.less` already handles appropriate display properties to children when open/closed.